shared-libs/outbound/test/outbound.spec.js

```diff
@@ -267,14 +268,15 @@ describe('outbound shared library', () => {
         }
       };
 
+      sinon.stub(environment, 'getVersion').resolves('4.18.0');
       sinon.stub(request, 'post').resolves();
 
       return outbound.__get__('sendPayload')(payload, conf)
         .then(() => {
           assert.equal(request.post.callCount, 1);
           assert.equal(request.post.args[0][0].url, 'http://test/foo');
           assert.deepEqual(request.post.args[0][0].body, {some: 'data'});
-          assert.equal(request.post.args[0][0].json, true);
+          assert.match(request.post.args[0][0].headers['user-agent'], /^CHT\/4\.18\.0/);
```

Comment:
Can you please add a test where os.platform and arch are also stubbed, so we see the complete user agent header as well?